### ISSUE TITLE
New Rule: disallow a use of both `exports.foo` and `module.exports` at same time

### ISSUE THREAD

**Issue by mysticatea** [mysticatea/eslint-plugin-node#50]
By default, `exports` is the same instance as `module.exports`.
But if you write `module.exports = {}`, the instance of those variables gets different.

This rule should warn the following code:

```js
module.exports = {
    foo() {}
}

// `bar` is not exported.
exports.bar = function() {
}
```

**Comment by mysticatea** [comment]
Fixed: 81ffd2554fcf95aff5224e66aca07bb0295fdd4e
